corehq/motech/repeaters/models.py

```diff
@@ -473,6 +479,14 @@ def send_request(self, repeat_record, payload):
             method=self.request_method,
         )
 
+    def do_request_preflight_check(self):
+        if self.connection_settings.is_throttled_connection:
+            if self.connection_settings.request_is_throttled():
```

Comment:
What is the difference between the conditional on this line and the one just above? It feels like we only need one of them as they both seem to be checking the same thing.

Reply:
I agree, but because there's some logic that needs to execute when `is_throttled_connection` is false, I can't simply rely on the result of `request_is_throttled`. In addition to this, it feels weird to have `request_is_throttled` without checking `is_throttled_connection`. 

If I had to change something, I'd rather leave `is_throttled_connection` out of `request_is_throttled`.

Reply:
I'm going to remove the check from `request_is_throttled`.